numpy/core/code_generators/generate_umath.py

```diff
@@ -1028,8 +1028,10 @@ def make_arrays(funcdict):
                 funclist.append('NULL')
                 try:
                     thedict = arity_lookup[uf.nin, uf.nout]
-                except KeyError:
-                    raise ValueError("Could not handle {}[{}]".format(name, t.type))
+                except KeyError as e:
+                    raise ValueError(
+                        "Could not handle {}[{}]".format(name, t.type)
```

Comment:
```suggestion
                        "Could not handle {}[{}] with nin={}, nout={}".format(name, t.type, uf.nin, uf.nout)
```
If we're going to drop the KeyError in the traceback, let's make sure this error contains the useful bit of info